rfcs/proposed/range_algorithms/README.md

```diff
@@ -0,0 +1,35 @@
+# Support the second portion of the oneDPL Range APIs
+
+## Introduction
+Based on statistics (observing C++ code within github.com) for the usage of popular algorithms, the following
+range-based APIs are suggested to be implemented next in oneDPL.
+`fill`, `generate`, `move`, `replace`, `replace_if`, `remove`, `remove_if`, `mismatch`, `minmax_element`, `minmax`,
+`min`, `max`, `find_first_of`, `find_end`, `is_sorted_until`
+
+## Motivations
+The feature is proposed as the next step of range-based API support for oneDPL.
+
+### Key Requirements
+- The range-based signatures for the mentioned API should correspond to the proposal for C++ parallel range algorithms, P3179.
+(https://wg21.link/p3179), but the all input and output ranges should be sized and random access ranges.
```

Comment:
I think here we can now refer to the specification update PR https://github.com/uxlfoundation/oneAPI-spec/pull/614
```suggestion
- The range-based signatures for the mentioned API should correspond to the [proposed specification](https://github.com/uxlfoundation/oneAPI-spec/pull/614)
that is based on the [C++ standardization proposal P3179](https://wg21.link/p3179).
```